package.json

```diff
@@ -40,11 +40,15 @@
     "whatwg-fetch": "^3.0.0"
   },
   "devDependencies": {
+    "@svgr/webpack": "^6.2.1",
     "eslint": "^5.16.0",
     "eslint-plugin-import": "^2.17.2",
     "eslint-plugin-react": "^7.13.0",
     "redux-devtools-extension": "^2.13.8"
   },
+  "overrides": {
+    "@svgr/webpack": "$@svgr/webpack"
+  },
   "scripts": {
```

Comment:
You may also want to add the node version requirements

Reply:
Hi @tobsign, Added the minimum version requirements that are free of vulnerabilities. The last set of warnings that I am getting with even the node 18 and latest versions are given below. Could you please review and confirm if these need to be removed.

`npm WARN ERESOLVE overriding peer dependency
npm WARN deprecated stable@0.1.8: Modern JS already guarantees Array#sort() is a stable sort, so this library is deprecated. See the compatibility table on MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#browser_compatibility
npm WARN deprecated rollup-plugin-terser@7.0.2: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-terser
npm WARN deprecated w3c-hr-time@1.0.2: Use your platform's native performance.now() and performance.timeOrigin.
npm WARN deprecated sourcemap-codec@1.4.8: Please use @jridgewell/sourcemap-codec instead
npm WARN deprecated redux-devtools-extension@2.13.9: Package moved to @redux-devtools/extension.
npm WARN deprecated @fortawesome/fontawesome@1.1.8: This package is deprecated. See https://git.io/fNCzJ for information about upgrading.
npm WARN deprecated @fortawesome/fontawesome-free-regular@5.0.13: This package is deprecated. See https://git.io/fNCzJ for information about upgrading.
npm WARN deprecated @fortawesome/fontawesome-free-solid@5.0.13: This package is deprecated. See https://git.io/fNCzJ for information about upgrading.`

Reply:
@sheetalkadwal were you able to identify where these dependencies are used in our code, and if we can't apply the fixes suggested in these warning messages? Thanks

Reply:
Hi @tobsign, Yes, these warnings are after other fixes. Fonts are getting used in ActiveTransfer.js. Seems like the icons used there are deprecated. Though the app is working as intended. I tried with the fonts for a while earlier but have been getting issues with the new suggested fonts, so I suspect it may take some time to get them fixed.